x-pack/docs/en/settings/security-settings.asciidoc

```diff
@@ -1225,6 +1225,13 @@ Password to the truststore.
 `xpack.ssl.truststore.secure_password` (<<secure-settings,Secure>>)::
 Password to the truststore.
 
+Note that storing trusted certificates in a PKCS#12 keystore, although supported,
+is uncommon in practice. Usually, keystore files only contain secret and private
```

Comment:
I think the use of `keystore files` is confusing here.
This is not true of JKS, nor is it true for PKCS#12 files that are managed by `keytool`

I think we need to say `PKCS#12 files` instead.